pytest_hot_reloading/plugin.py

```diff
@@ -98,8 +142,10 @@ def _plugin_logic(session: Session) -> None:
         # pytest prints out "collecting ...". The leading \r prevents that
         print("\rStarting daemon...")
         pattern = _get_pattern_filters(session)
-        jurigged.watch(pattern=pattern, logger=_jurigged_logger)
+        # TODO: intelligently use poll versus watchman
+        jurigged.watch(pattern=pattern, logger=_jurigged_logger, poll=True)
```

Comment:
In codespaces watchman worked while on a dev container on my windows machine it did not detect changes. Polling has high CPU usage in large repos so watchman is certainly preferred. Added #16 to revisit this